cherrypy/_cperror.py

```diff
@@ -271,7 +271,7 @@ def set_response(self):
                 307: 'This resource has moved temporarily to ',
             }[status]
             msg += '<a href=%s>%s</a>.'
-            msgs = [msg % (saxutils.quoteattr(u), u) for u in self.urls]
+            msgs = [msg % (saxutils.quoteattr(u), escape_html(u)) for u in self.urls]
```

Comment:
I just think that using several different libs for doing the same thing. Moreover, current implementation does conversion of the **exactly same** variable during each loop iteration. It's much more optimal to do this one time.